aptos-move/e2e-move-tests/src/tests/randomness.data/pack/sources/test.move

```diff
@@ -17,7 +17,8 @@ module 0x1::test {
         let _ = randomness::u64_integer();
     }
 
-    entry fun fail_if_not_annotated_and_using_randomness() {
+    #[lint_allow_unsafe_randomness]
```

Comment:
I am assuming this is to test that an arbitrary attribute (such as the now unsupported  `#[lint_allow_unsafe_randomness]`) still causes failure?

Or was this supposed to be updated to `#[lint::allow_unsafe_randomness]`?

Reply:
Oversight, this fails the test. Fixed.